src/rad/resources/schemas/exposure-1.1.0.yaml

```
@@ -0,0 +1,246 @@
+      were transmitted to the ground.
+    type: integer
+    sdf:
+      special_processing: VALUE_REQUIRED
+      source:
+        origin: PSS:dms_visit.multi_accum_resultant
+    archive_catalog:
+      datatype: int
+      destination: [WFIExposure.exposure_nresultants, GuideWindow.exposure_nresultants, WFICommon.exposure_nresultants]
+  data_problem:
+    title: Data Problem
+    description: |
+      String error codes indicating that a problem occurred with the exposure.
+      A value of None indicates there were no problems with the exposure.
+      See the Roman Documentation System (RDox) for more information on the meaning of the error codes.
+    anyOf:
+      - type: string
+      - type: "null"
+    sdf:
+      special_processing: VALUE_REQUIRED
+      source:
+        origin: TBD
+    archive_catalog:
+      datatype: nvarchar(max)
+      destination: [WFIExposure.exposure_data_problem, GuideWindow.exposure_data_problem, WFICommon.exposure_data_problem]
+  frame_time:
+    title: Detector Readout Time (s)
+    description: |
+      The readout time in seconds of the Wide Field
+      Instrument (WFI) detectors. This represents the time between the
+      start and end of the readout.
+    type: number
+    sdf:
+      special_processing: VALUE_REQUIRED
+      source:
+        origin: TBD
+    archive_catalog:
+      datatype: float
+      destination: [WFIExposure.exposure_frame_time, GuideWindow.exposure_frame_time, WFICommon.exposure_frame_time]
+  exposure_time:
+    title: Exposure Time (s)
+    description: |
+      The difference in time (in units of seconds) between
+      the start of the first reset/read and end of the last read in
+      the readout pattern.
+    type: number
+    sdf:
+      special_processing: VALUE_REQUIRED
+      source:
+        origin: TBD
+    archive_catalog:
+      datatype: float
+      destination: [WFIExposure.exposure_time, GuideWindow.exposure_time, WFICommon.exposure_time]
+  effective_exposure_time:
+    title: Effective Exposure Time (s)
+    description: |
+      The difference in time (in units of seconds) between
+      the midpoints of the first and last science resultants. If
+      either resultant contains only a single read, then the time at
+      the end of the read is used rather than the midpoint time.
+    type: number
+    sdf:
+      special_processing: VALUE_REQUIRED
+      source:
+        origin: TBD
+    archive_catalog:
+      datatype: float
+      destination: [WFIExposure.effective_exposure_time, GuideWindow.effective_exposure_time,
+                    SourceCatalog.effective_exposure_time]
+  ma_table_name:
+    title: Name of the Multi-Accumulation Table
+    description: |
+      The name of the multi-accumulation (MA) table used for
+      the exposure. The MA table is also referred to as the readout
+      pattern.
+    type: string
+    sdf:
+      special_processing: VALUE_REQUIRED
+      source:
+        origin: PSS:dms_visit.multi_accum_table_name
```

```suggestion
        origin: PSS:dms_visit.multi_accum_table_name
```
Is this change expected? It doesn't match what's in `exposure-1.0.0`:
https://github.com/spacetelescope/rad/blob/7147612732b4578d0ec03ebcf90ce592e9312941/src/rad/resources/schemas/exposure-1.0.0.yaml#L149-L159

---

Or was this left out of #571?

---

It was accidentally left out of #571. I spoke with the user (RSDP) and confirmed that it should be PSS:dms_visit.multi_accum_table_name